OpenEdXMobile/src/main/java/org/edx/mobile/view/dialog/RatingDialogFragment.java

```diff
@@ -39,6 +38,8 @@ public class RatingDialogFragment extends DialogFragment implements AlertDialog.
     Router mRouter;
     @Inject
     AnalyticsRegistry analyticsRegistry;
+    @Inject
```

Comment:
Please add a new line.

Reply:
we didn't add spaces in above 2 Injections as well, so that's why put it together.